inobounce.js

```diff
@@ -22,6 +22,11 @@
 				break;
 			}
 
+            //ignore range input element
```

Comment:
Nitpick: Space after `//` and capitalize first letter of the comment to match commenting style elsewhere